docs/zh_cn/index.rst

```diff
@@ -20,6 +20,7 @@
    02-how-to-run/convert_model.md
    02-how-to-run/write_config.md
    02-how-to-run/profile_model.md
+   02-how-to-run/quantize_model.md
```

Comment:
`03-benchmark/quantization.md` can be added in index.rst too.

Reply:
done.